packages/grafana-schema/src/common/common.gen.ts

```diff
@@ -995,6 +995,10 @@ export interface TableFieldOptions {
   hideHeader?: boolean;
   inspect: boolean;
   minWidth?: number;
+  /**
+   * Selecting or hovering this field will show a tooltip containing the content within the target field
+   */
+  tooltipByField?: string;
```

Comment:
me may want to nest this into `tooltip.field` or something, i imagine we may want quite a few more tooltip opts.